[4.0] atum Toolbar items sizing issue

himanshu007-creator opened the pull request:
Pull Request for Issue #34109 .

### Summary of Changes
On enabling Highlight Links in accessibility, there was mismatch in size of toolbar items because of additional padding.
But keeping accessibility in mind, other things should be left unchanged, therefore just removed the padding.


### Testing Instructions
- apply the patch and rebuild css <code>npm run build:css</code>
- turn highlight links on from accessibility and note, size doesnt change

### Actual result BEFORE applying this Pull Request
<img width="711" alt="old" src="https://user-images.githubusercontent.com/65963997/119274356-19f57900-bc2d-11eb-8a44-dc867eff3443.png">

### Expected result AFTER applying this Pull Request
![ssss](https://user-images.githubusercontent.com/65963997/119296017-cbb89800-bc75-11eb-83f1-5a34f5ba5bda.gif)




### Documentation Changes Required

himanshu007-creator committed:
Update _global.scss

himanshu007-creator committed:
lint

The following pattern is not considered a violation:
a { --x: 0px; }

himanshu007-creator committed:
Update _global.scss

PhilETaylor commented:
I think you misunderstood the issue. 

The issue is 

1)  the User Menu oval background is not the same height and depth as the other two (Postinstallation messages and joomla4 in your screenshot) 

2) User Menu is not underlined.

himanshu007-creator committed:
usermenu changed

made usermenu style consistent with others and removed styling from joomlaversion

himanshu007-creator commented:
@PhilETaylor made the changes as suggested by you.

PhilETaylor commented:
Looks good - thanks.

<img width="1010" alt="Screenshot 2021-05-25 at 20 06 57" src="https://user-images.githubusercontent.com/400092/119554597-c56d0d80-bd94-11eb-8ebd-606d37755f1a.png">

PhilETaylor commented:
I have tested this item :white_check_mark: successfully on f7dedbce4dd61721db7bc1ff1674cf4a4e5c885f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34146">issues.joomla.org/tracker/joomla-cms/34146</a>.</sub>

sandramay0905 commented:
I have tested this item :white_check_mark: successfully on f7dedbce4dd61721db7bc1ff1674cf4a4e5c885f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34146">issues.joomla.org/tracker/joomla-cms/34146</a>.</sub>

richard67 commented:
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34146">issues.joomla.org/tracker/joomla-cms/34146</a>.</sub>

Quy commented on administrator/templates/atum/scss/blocks/_global.scss:
```diff
@@ -48,6 +48,18 @@ body {
       border: #f00 dotted 1px;
     }
+
+    .header-item-content {
```
Please combine with the above `a` since attributes are the same.

himanshu007-creator replied:
Okay

himanshu007-creator committed:
Update _global.scss

himanshu007-creator commented:
Made the changes @richard67

Quy commented on administrator/templates/atum/scss/blocks/_global.scss:
```diff
@@ -43,11 +43,17 @@ body {
 
   &.a11y_highlight {
-    a {
+    a, .header-item-content {
```
```suggestion
    a,
    .header-item-content {
```

himanshu007-creator committed:
Update administrator/templates/atum/scss/blocks/_global.scss

Co-authored-by: Quy <quy@fluxbb.org>

Quy commented on administrator/templates/atum/scss/blocks/_global.scss:
```diff
@@ -43,11 +43,18 @@ body {
+
+    .joomlaversion {
+      padding: 0;
+      text-decoration: none;
```
```suggestion
      text-decoration: none !important;
```

Quy replied:
Please confirm as my previous commit added underline to the version. Thanks.

himanshu007-creator committed:
Update administrator/templates/atum/scss/blocks/_global.scss

Co-authored-by: Quy <quy@fluxbb.org>

sandramay0905 commented:
I have tested this item :white_check_mark: successfully on 9d2c6305570314ecbd3bbfd8d702a72cdd904c5e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34146">issues.joomla.org/tracker/joomla-cms/34146</a>.</sub>

srishty-07 commented:
I have tested this item :white_check_mark: successfully on 9d2c6305570314ecbd3bbfd8d702a72cdd904c5e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34146">issues.joomla.org/tracker/joomla-cms/34146</a>.</sub>

Quy commented:
Thanks!
